src/app/components/home/twitter-feed/twitter-feed.component.spec.ts

```diff
@@ -3,15 +3,17 @@ import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { TwitterFeedComponent } from './twitter-feed.component';
 import { WindowService } from '../../../services/window.service';
 import { NO_ERRORS_SCHEMA } from '@angular/core';
+import {MockWindowService} from '../../../services/mock.window.service';
 
 describe('TwitterFeedComponent', () => {
   let component: TwitterFeedComponent;
   let fixture: ComponentFixture<TwitterFeedComponent>;
 
   beforeEach(async(() => {
+    const MOCK_SERVICE = new MockWindowService(null);
     TestBed.configureTestingModule({
       declarations: [ TwitterFeedComponent ],
-      providers: [ WindowService ],
+      providers: [ {provide: WindowService, useValue: MOCK_SERVICE } ],
```

Comment:
@sanketbansal could we add some more specs (other than the default ones) here now or you'll add them in the already open PR?

Reply:
I will add test cases in other open PRs. I created this PR as other PR are dependent over this PR.